checkov/sca_package/scanner.py

```diff
@@ -14,6 +14,8 @@
 
 SLEEP_DURATION = 2
 MAX_SLEEP_DURATION = 60
+REQUEST_MAX_TRIES = os.getenv('REQUEST_MAX_TRIES', 2)
+REQUEST_SLEEP_BETWEEN_TRIES = os.getenv('REQUEST_SLEEP_BETWEEN_TRIES', 0.1)
```

Comment:
@itai1357 Please use `CHECKOV_` as the prefix of every env var we use, and set base retry to 3
Also, shouldn't these be `int`s?

Reply:
yep you are right!